loky/backend/popen_loky_posix.py

```diff
@@ -194,10 +192,10 @@ def thread_is_spawning():
         import msvcrt
```

Comment:
We can remove this as we are in the `popen_loky_posix.py` file which is only run on POSIX platforms